src/main/java/ch/njol/skript/effects/EffUnlimitedLifetime.java

```diff
@@ -0,0 +1,68 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Item;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Unlimited Lifetime")
+@Description("Makes a dropped item have an unlimited lifetime.")
+@Examples({
+	"drop tripwire hook at player's location",
+	"make last dropped item have unlimited lifetime"})
+public class EffUnlimitedLifetime extends Effect {
+
+	static {
+		Skript.registerEffect(EffUnlimitedLifetime.class,
+			"make %entities% have [an] (unlimited|infinite) life(time|span)");
```

Comment:
```suggestion
		Skript.registerEffect(EffUnlimitedLifetime.class, "make %entities% have [an] (unlimited|infinite) life(time|span)");
```
Do this first one if it's not too long, or do:
```suggestion
		Skript.registerEffect(EffUnlimitedLifetime.class,
			"make %entities% have [an] (unlimited|infinite) life(time|span)"
		);
```